### DISCUSSION TITLE
Exception traceback when live logs are enabled

### DISCUSSION THREAD

**Discussion by Sup3rGeo** [pytest-dev/pytest#8240] [Ideas] [answered]
Hi, just a quick question I couldn't manage to find it myself.

Normally we run with the live logs enabled, which is super cool as we follow the setup, call and teardown per test according to the execution.

However, one thing that is inconvenient is that the exception traceback is still shown at the end of the session for all tests, whereas, in our case, it would be much more useful to have it already within the specific test real time logs as well:

```
test_one
  -- live log setup --
  ...
  -- live log call --
  ...
  -- Exception traceback --
  ...
  -- live log teardown --
  ....

test_two
  -- live log setup --
```

and so on

Can it be already done with a specific option? If not, would it be an interesting feature? I would be happy to implement it and open a PR for that.

Thanks!

**Comment by nicoddemus** [comment, marked as answer]
Hey @Sup3rGeo!

Try out [pytest-instafail](https://pypi.org/project/pytest-instafail/), might be what you are looking for. 👍

**Reply by Sup3rGeo** [reply]
Exactly! Thanks so much
